packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -86,7 +86,26 @@ The following demo shows how to customize the appointment form. A custom `TextEd
 
 ## Add a Confirmation Dialog
 
-
-To enable the confirmation dialog, add the [ConfirmationDialog](../reference/confirmation-dialog.md) plugin. The dialog pops up when a user attempts to delete an appointment or discard edits made to it. If the dialog should not appear in these cases, set the `ignoreDelete` or `ignoreCancel` property to `true`. 
+To enable the confirmation dialog, add the [ConfirmationDialog](../reference/confirmation-dialog.md) plugin. The dialog pops up when a user attempts to delete an appointment or discard edits made to it. If the dialog should not appear in these cases, set the `ignoreDelete` or `ignoreCancel` property to `true`.
 
 .embedded-demo({ "path": "scheduler-editing/delete-confirmation", "showThemeSelector": true })
+
+## Disable one or several editing options
+
+By default, if you add the [EditingState](../reference/editing-state.md), [AppointmentForm](../reference/appointment-form.md), [IntegratedEditing](../reference/integrated-editing.md) or [EditRecurrenceMenu](../reference/integrated-editing.md),  and [DragDropProvider](../reference/drag-drop-provider.md) plugins, all editing options are turned on. But they can be disabled. Below are the steps to disable them:
```

Comment:
Excess paragraph. We should write less but enough to understand how to do something.